### PR TITLE
adding xanderyzwich to participants.md

### PR DESCRIPTION
**Pull request by xanderyzwich**
Users formatted as bullet list
Note added about ordering
added self

### CODE REVIEW

**Review by ericwburden — CHANGES_REQUESTED** [review]
I think alphabetizing by last name will make it more difficult to quickly scan, since we're listing names in 'First Last' format. I feel like alphabetizing by first name will be more intuitive here.

**Review by parhaml — DISMISSED** [review]
Nice changes - Acutally I see what @ericwburden. What should the stragegy be? Alphabetize First name?

File: participants.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # CodeConnector CodingDojo Participants
 ---
 
-This file will act as a short exercise in [contributing](https://guides.github.com/activities/forking/) to the CodingDojo exercises.  You'll fork this repo and add your name to this file.
+This file will act as a short exercise in [contributing](https://guides.github.com/activities/forking/) to the CodingDojo exercises.  You'll fork this repo and add your name to this file. Names should be added in alphabetical order by last name.
```
Comment (drkennetz) [inline participants.md:5]: I think we should alphabetize by first name as well.

**Review by drkennetz — CHANGES_REQUESTED** [review]

**Comment by xanderyzwich** [comment]
@drkennetz @ericwburden @parhaml 
Change has been made to be ordered by first name (oddly enough this did not change the list only the description)

File: participants.md (hunk 1 above)
Comment (xanderyzwich) [reply participants.md:5]: done

**Review by parhaml — APPROVED** [review]

**Review by ericwburden — APPROVED** [review]

**Pull request merged**